R/install_livy.R

```diff
@@ -0,0 +1,70 @@
+#' Install Livy
+#'
+#' Automatically download and install \href{http://livy.io/}{\samp{livy}}.
+#' \samp{livy} provides a REST API to Spark.
+#'
+#' @param spark_home The path to a Spark installation. The downloaded and
+#'   installed version of \samp{livy} will then be associated with this Spark
+#'   installation.
+#' @param livy_version The version of \samp{livy} to be installed. Currently,
+#'   only the beta version, 0.2.0, is available.
+#' @param hadoop_conf_dir Optional; the path to a Hadoop configuration directory.
+#' @export
+livy_install <- function(spark_home = Sys.getenv("SPARK_HOME"),
+                         livy_version = "0.2.0",
+                         hadoop_conf_dir = NULL)
+{
+  hadoop_conf_dir <- ensure_scalar_character(hadoop_conf_dir, default = "")
+
+  # ensure that spark home exists
+  if (!file.exists(spark_home))
+    stopf("No Spark installation found at '%s'", spark_home)
+
+  # use the directory path as the version
```

Comment:
We should add a check, since I just verified that Livy `0.2.0` does not support Spark `2.0.0`. See this mail thread: https://groups.google.com/a/cloudera.org/forum/#!topic/livy-dev/50qlsi-g23I.